chart/templates/configmap.yaml

```diff
@@ -11,4 +11,6 @@ data:
   default_resource_requests_memory: "128Mi"
   always_send_finished_event: "false"
   enable_kubernetes_api_access: "{{ .Values.jobConfig.enableKubernetesApiAccess | default false }}"
+  keptn_api_endpoint: {{ (printf "%s://%s/api" .Values.remoteControlPlane.api.protocol .Values.remoteControlPlane.api.hostname) }}
```

Comment:
are we sure we are always going to use default protocol ports ? Or the port is to be included in the hostname ?

Reply:
Port is part of hostname